File: app/src/main/java/com/music/player/bhandari/m/equalizer/EqualizerSetting.java
package com.music.player.bhandari.m.equalizer;

/**
 * Copyright 2017 Amit Bhandari AB
 * <p>
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 * <p>
 * http://www.apache.org/licenses/LICENSE-2.0
 * <p>
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

public class EqualizerSetting {

    private int fiftyHertz = 16;
    private int oneThirtyHertz = 16;
    private int threeTwentyHertz = 16;
    private int eightHundredHertz = 16;
    private int twoKilohertz = 16;
    private int fiveKilohertz = 16;
    private int twelvePointFiveKilohertz = 16;
    private int virtualizer = 16;
    private int bassBoost = 16;
    private int enhancement = 16;
    private int reverb = 16;

    public EqualizerSetting(int fiftyHertz, int oneThirtyHertz, int threeTwentyHertz, int eightHundredHertz, int twoKilohertz, int fiveKilohertz, int twelvePointFiveKilohertz
            , int virtualizer, int bassBoost, int reverb) {
        this.fiftyHertz = fiftyHertz;
        this.oneThirtyHertz = oneThirtyHertz;
        this.threeTwentyHertz = threeTwentyHertz;
        this.eightHundredHertz = eightHundredHertz;
        this.twoKilohertz = twoKilohertz;
        this.fiveKilohertz = fiveKilohertz;
        this.twelvePointFiveKilohertz = twelvePointFiveKilohertz;
        this.virtualizer = virtualizer;
        this.bassBoost = bassBoost;
        this.reverb = reverb;
    }

    public EqualizerSetting() {
    }

    @Override
    public String toString() {
        return "" + fiftyHertz + " : "
                + oneThirtyHertz + " : "
                + threeTwentyHertz + " : "
                + eightHundredHertz + " : "
                + twoKilohertz + " : "
                + fiveKilohertz + " : "
                + twelvePointFiveKilohertz + " : "
                + virtualizer + " : "
                + bassBoost + " : "
                + reverb + " : ";
    }

    public int getFiftyHertz() {
        return fiftyHertz;
    }

    public void setFiftyHertz(int fiftyHertz) {
        this.fiftyHertz = fiftyHertz;
    }

    public int getOneThirtyHertz() {
        return oneThirtyHertz;
    }

    public void setOneThirtyHertz(int oneThirtyHertz) {
        this.oneThirtyHertz = oneThirtyHertz;
    }

    public int getThreeTwentyHertz() {
        return threeTwentyHertz;
    }

    public void setThreeTwentyHertz(int threeTwentyHertz) {
        this.threeTwentyHertz = threeTwentyHertz;
    }

    public int getEightHundredHertz() {
        return eightHundredHertz;
    }

    public void setEightHundredHertz(int eightHundredHertz) {
        this.eightHundredHertz = eightHundredHertz;
    }

    public int getTwoKilohertz() {
        return twoKilohertz;
    }

    public void setTwoKilohertz(int twoKilohertz) {
        this.twoKilohertz = twoKilohertz;
    }

    public int getFiveKilohertz() {
        return fiveKilohertz;
    }

    public void setFiveKilohertz(int fiveKilohertz) {
        this.fiveKilohertz = fiveKilohertz;
    }

    public int getTwelvePointFiveKilohertz() {
        return twelvePointFiveKilohertz;
    }

    public void setTwelvePointFiveKilohertz(int twelvePointFiveKilohertz) {
        this.twelvePointFiveKilohertz = twelvePointFiveKilohertz;
    }

    public int getVirtualizer() {
        return virtualizer;
    }

    public void setVirtualizer(int virtualizer) {
        this.virtualizer = virtualizer;
    }

    public int getBassBoost() {
        return bassBoost;
    }

    public int getEnhancement() {
        return enhancement;
    }

    public void setBassBoost(int bassBoost) {
        this.bassBoost = bassBoost;
    }

    public void setEnhancement(int enhancement) {
        this.enhancement = enhancement;
    }

    public int getReverb() {
        return reverb;
    }

    public void setReverb(int reverb) {
        this.reverb = reverb;
    }
}

File: app/src/main/java/com/music/player/bhandari/m/equalizer/EqualizerSetting.kt
package com.music.player.bhandari.m.equalizer

/**
 * Copyright 2017 Amit Bhandari AB
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
class EqualizerSetting {
    private var fiftyHertz: Int = 16
    private var oneThirtyHertz: Int = 16
    private var threeTwentyHertz: Int = 16
    private var eightHundredHertz: Int = 16
    private var twoKilohertz: Int = 16
    private var fiveKilohertz: Int = 16
    private var twelvePointFiveKilohertz: Int = 16
    private var virtualizer: Int = 16
    private var bassBoost: Int = 16
    private var enhancement: Int = 16
    private var reverb: Int = 16

    constructor(
        fiftyHertz: Int,
        oneThirtyHertz: Int,
        threeTwentyHertz: Int,
        eightHundredHertz: Int,
        twoKilohertz: Int,
        fiveKilohertz: Int,
        twelvePointFiveKilohertz: Int,
        virtualizer: Int,
        bassBoost: Int,
        reverb: Int
    ) {
        this.fiftyHertz = fiftyHertz
        this.oneThirtyHertz = oneThirtyHertz
        this.threeTwentyHertz = threeTwentyHertz
        this.eightHundredHertz = eightHundredHertz
        this.twoKilohertz = twoKilohertz
        this.fiveKilohertz = fiveKilohertz
        this.twelvePointFiveKilohertz = twelvePointFiveKilohertz
        this.virtualizer = virtualizer
        this.bassBoost = bassBoost
        this.reverb = reverb
    }

    constructor()

    override fun toString(): String {
        return ("" + fiftyHertz + " : "
                + oneThirtyHertz + " : "
                + threeTwentyHertz + " : "
                + eightHundredHertz + " : "
                + twoKilohertz + " : "
                + fiveKilohertz + " : "
                + twelvePointFiveKilohertz + " : "
                + virtualizer + " : "
                + bassBoost + " : "
                + reverb + " : ")
    }

    fun getFiftyHertz(): Int {
        return fiftyHertz
    }

    fun setFiftyHertz(fiftyHertz: Int) {
        this.fiftyHertz = fiftyHertz
    }

    fun getOneThirtyHertz(): Int {
        return oneThirtyHertz
    }

    fun setOneThirtyHertz(oneThirtyHertz: Int) {
        this.oneThirtyHertz = oneThirtyHertz
    }

    fun getThreeTwentyHertz(): Int {
        return threeTwentyHertz
    }

    fun setThreeTwentyHertz(threeTwentyHertz: Int) {
        this.threeTwentyHertz = threeTwentyHertz
    }

    fun getEightHundredHertz(): Int {
        return eightHundredHertz
    }

    fun setEightHundredHertz(eightHundredHertz: Int) {
        this.eightHundredHertz = eightHundredHertz
    }

    fun getTwoKilohertz(): Int {
        return twoKilohertz
    }

    fun setTwoKilohertz(twoKilohertz: Int) {
        this.twoKilohertz = twoKilohertz
    }

    fun getFiveKilohertz(): Int {
        return fiveKilohertz
    }

    fun setFiveKilohertz(fiveKilohertz: Int) {
        this.fiveKilohertz = fiveKilohertz
    }

    fun getTwelvePointFiveKilohertz(): Int {
        return twelvePointFiveKilohertz
    }

    fun setTwelvePointFiveKilohertz(twelvePointFiveKilohertz: Int) {
        this.twelvePointFiveKilohertz = twelvePointFiveKilohertz
    }

    fun getVirtualizer(): Int {
        return virtualizer
    }

    fun setVirtualizer(virtualizer: Int) {
        this.virtualizer = virtualizer
    }

    fun getBassBoost(): Int {
        return bassBoost
    }

    fun getEnhancement(): Int {
        return enhancement
    }

    fun setBassBoost(bassBoost: Int) {
        this.bassBoost = bassBoost
    }

    fun setEnhancement(enhancement: Int) {
        this.enhancement = enhancement
    }

    fun getReverb(): Int {
        return reverb
    }

    fun setReverb(reverb: Int) {
        this.reverb = reverb
    }
}